src/components/read-along-component/read-along.tsx

```diff
@@ -636,15 +774,33 @@ export class ReadAlongComponent {
   returnTranslation(word: string, lang: string): string {
```

Comment:
Please update the documentation above to note that "eng" has changed to "en".

Reply:
I think it is better to stick to eng and fra iso-639-3 code standards. Refactored to support eng and fra. 
Added code to handle backwards compatibility.